pkg/webhook/server/server.go

```diff
@@ -33,13 +33,13 @@ type ServerModifier func(w *webhook.Server)
 
 // Add creates a new Conditions Mutating Webhook and adds it to the Manager. The Manager will set fields on the Webhook
 // and Start it when the Manager is Started.
-func New(client client.Client, webhookName string, webhookType certificate.WebhookType, serverOpts ...ServerModifier) *Server {
+func New(client client.Client, webhookName string, webhookType certificate.WebhookType, caRotateInterval time.Duration, serverOpts ...ServerModifier) *Server {
```

Comment:
Should not we use ServerModifier for that?

Reply:
we don't want to make it optional.